client/profile-wizard/steps/business-details/flows/selective-bundle/index.js

```diff
@@ -48,39 +49,80 @@ export const filterBusinessExtensions = ( extensionInstallationOptions ) => {
 					extensionInstallationOptions[ key ] &&
 					key !== 'install_extensions'
 			)
-			.map( ( key ) => {
-				// Remove anything after :
-				// Please refer to selective-extensions-bundle/index.js
-				// installableExtensions variable
-				// this is to allow duplicate slugs (Tax & Shipping for example)
-				return key.split( ':' )[ 0 ];
-			} )
+			.map( getPluginSlug )
 			// remove duplicate
 			.filter( ( item, index, arr ) => arr.indexOf( item ) === index )
 	);
 };
 
+const timeFrames = [
+	{ name: '0-2s', max: 2 },
+	{ name: '2-5s', max: 5 },
+	{ name: '5-10s', max: 10 },
+	{ name: '10-15s', max: 15 },
+	{ name: '>15s' },
+];
+function getTimebox( timeInMs ) {
+	for ( const timeBox of timeFrames ) {
+		if ( ! timeBox.max ) {
+			return timeBox.name;
+		}
+		if ( timeInMs < timeBox.max * 1000 ) {
+			return timeBox.name;
+		}
+	}
+}
+
 export const prepareExtensionTrackingData = (
 	extensionInstallationOptions
 ) => {
 	const installedExtensions = {};
-	for ( const [ fieldKey, value ] of Object.entries(
+	for ( let [ fieldKey, value ] of Object.entries(
 		extensionInstallationOptions
 	) ) {
-		const key =
-			fieldKey === 'woocommerce-payments'
-				? 'install_wcpay'
-				: `install_${ fieldKey.replace( /-/g, '_' ).split( ':', 1 ) }`;
+		fieldKey = getPluginSlug( fieldKey );
+		const key = getPluginTrackKey( fieldKey );
 		if (
 			fieldKey !== 'install_extensions' &&
-			! installedExtensions[ key ]
+			! installedExtensions[ `install_${ key }` ]
 		) {
-			installedExtensions[ key ] = value;
+			installedExtensions[ `install_${ key }` ] = value;
 		}
 	}
 	return installedExtensions;
 };
 
+export const prepareExtensionTrackingInstallationData = (
+	extensionInstallationOptions,
+	installationData
+) => {
+	const installed = [];
+	const data = {};
+	for ( let [ fieldKey ] of Object.entries( extensionInstallationOptions ) ) {
+		fieldKey = getPluginSlug( fieldKey );
+		const key = getPluginTrackKey( fieldKey );
+		if (
```

Comment:
I believe this condition is not passing for me.  I'm not getting installation data or times in the latest version.

![Screen Shot 2021-12-27 at 3 09 45 PM](https://user-images.githubusercontent.com/10561050/147504071-ed72ac85-e721-4037-bf6f-dfba7c2eba1c.png)

Reply:
@joshuatf is this because these plugins were already installed and just needed to be activated?
This condition is triggered for me, when an installation is required (this does depend on changes in the data package, so maybe that didn't get updated?)

Reply:
That was with plugins completely removed.  I can try again later today and rebuild all packages to see if that makes a difference.